versions.properties

```diff
@@ -46,3 +46,6 @@ version.org.jetbrains.kotlinx..binary-compatibility-validator=0.2.3
 version.org.jlleitschuh.gradle..ktlint-gradle=9.2.1
 version.org.openjdk.jmh..jmh-core=1.25.2
 version.org.openjdk.jmh..jmh-generator-annprocess=1.25.2
+
+version.androidx.test.core=1.3.0
+##             # available=1.3.1-alpha03
```

Comment:
?

Reply:
My response exactly. Will get rid of this.